app/models/maintenance_stats/stats/github/commit_count_stats.rb

```diff
@@ -0,0 +1,61 @@
+module MaintenanceStats
+  module Stats
+    module Github
+      class BaseCommitCount < BaseStat
+        def pull_out_commit_count(dataset)
+            return nil if dataset.data.repository.nil? || dataset.data.repository.default_branch_ref.nil?
+            dataset.data.repository.default_branch_ref.target.history.total_count
+        end
+      end
+
+      class LastWeekCommitsStat < BaseCommitCount
```

Comment:
Out of curiosity, why classes with a single method in them rather than a bunch of methods?

Reply:
I think it was initially in the way I set up the method that took queries and assigned them to stat classes. That was changed before merge to be a little more simple. I would like to run back through the Github pieces tomorrow and get those queries and stat classes cleaned up.